ui/shared/status/StatusIcon.qml

```diff
@@ -0,0 +1,29 @@
+import QtQuick 2.13
+import QtGraphicalEffects 1.13
+
+Image {
```

Comment:
Should this be `SVGImage`?

Reply:
Great question!
I believe we'll remove `SVGImage` in the midterm. Cause all it does is setting source width and height (for which it needs `height` and `width` otherwise it causes binding loops). Most of the usages of `SVGImage` can be replaced by `StatusIcon` anyways.